packages/components/src/components/Table/Table.stories.tsx

```diff
@@ -49,6 +49,38 @@ Default.parameters = {
   },
 };
 
+export const StickyHeaders = (): JSX.Element => {
+  return (
+    <Table>
+      <THead stickyHeader>
```

Comment:
I started the storybook on my local machine and the head is not sticky. Can you check this, please?

Reply:
It seems to be working locally for me. Are you looking in a specific browser?

Reply:
I'm using Chrome.

Reply:
And also I could see the error in the storybook URL that Chromatic generates. 🤔 https://632b1baf1257171b6a84f370-qkgatvyujm.chromatic.com/?path=/story/components-anchor--default

Reply:
I see now. Its in the docs. Fix incoming.